lib/modules/manager/sbt/extract.ts

```diff
@@ -36,6 +38,10 @@ interface Ctx {
 
 const scala = lang.createLang('scala');
 
+const sbtVersionRegex = regEx(
+  '\\s*sbt\\.version\\s*=\\s*(\\d*\\.\\d*\\.\\d*)\\s*'
```

Comment:
```suggestion
  '\\s*sbt\\.version\\s*=\\s*(?<version>\\d*\\.\\d*\\.\\d*)\\s*'
```
use named capture group.

Reply:
The regex is a bit "over-matching" and also captures invalid versions like `1..` or `1.6.`

Suggestion:

```suggestion
  '^sbt\\.version\\s*[= :]\\s*(?<version>\\d+\\.\\d+\\.\\d+)\\s*$'
```

Reply:
I tried to be sure but sbt only supports `=` as delimiter, otherwise it will overwrite it and replace with `=`, thus I think we can go we with only supporting `=`.